drivers/ansible_shell/driver.py

```diff
@@ -15,4 +16,9 @@ def initialize(self, context):
     def execute_playbook(self, context, ansible_configuration_json, cancellation_context):
         return self.ansible_shell.execute_playbook(context, ansible_configuration_json, cancellation_context)
 
+    def execute_playbooks(self, context, ansible_configurations_json, cancellation_context):
+        configurations = json.loads(ansible_configurations_json)
```

Comment:
What happens if the seralization process fails?

Reply:
you get an exception - which is a good thing since the caller would be able to inspect it